pyndl/__init__.py

```diff
@@ -17,7 +17,7 @@
 __author__ = ('Konstantin Sering, Marc Weitz, '
               'David-Elias Künstle, Lennard Schneider')
 __author_email__ = 'konstantin.sering@uni-tuebingen.de'
-__version__ = '0.5.1'
+__version__ = '0.5.2'
```

Comment:
If the code breaks an existing api you should change major version to follow semver.
Or at least keep the old api which calls the new one internally + printing an deprecation warning.